### ISSUE TITLE
delay_factor throwing error

### ISSUE THREAD

**Issue by jedelman8** [networktocode/ntc-ansible#72]
hey @jtdub, did you test ntc_show_command after you added `delay_factor`? :)

I haven't used `netmiko` too much but just checked out the code and it looks like delay_factor is a config command and shouldn't be used in the ConnectHandler...

tests from netimiko:
```
 send_command_fast = net_connect.send_command(basic_command, delay_factor=.25)
```

```
failed: [spine2] => {"failed": true, "parsed": false}
Traceback (most recent call last):
  File "/home/cisco/.ansible/tmp/ansible-tmp-1446166372.68-170120052390508/ntc_show_command", line 1857, in <module>
    main()
  File "/home/cisco/.ansible/tmp/ansible-tmp-1446166372.68-170120052390508/ntc_show_command", line 223, in main
    password=password
  File "/usr/local/lib/python2.7/dist-packages/netmiko/ssh_dispatcher.py", line 54, in ConnectHandler
    return ConnectionClass(*args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'delay_factor'


FATAL: all hosts have already failed -- aborting

```

**Comment by jtdub** [comment]
You're right. It goes in send_command(). Standby.

**Comment by jtdub** [comment]
https://github.com/networktocode/ntc-ansible/pull/73

**Comment by jedelman8** [comment]
Fixed by #73
